io/net/virt-net/Network_virtualization_dlpar.py

```diff
@@ -198,8 +223,9 @@ def listhwres(self, server, lpar, slot_num):
             self.fail("lshwres operation failed ")
         return output
 
-    def changehwres(self, server, operation, lpar_id, slot_num,
-                    backing_devices, msg):
+    def Network_virtualization_add_device(self, server, operation,
```

Comment:
It should not have `add` in name, as it removes as well

Reply:
docstring missing

Reply:
Sure. Will make those changes